### PR TITLE
fix: remove ngrok from force dependencies

### PR DESCRIPTION
**Pull request by charan678**
# Fixes #

A short description of what this PR does.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,9 +78,6 @@
     "tildify": "^2.0.0",
     "tmp": "^0.2.1"
   },
-  "optionalDependencies": {
-    "ngrok": "^3.2.7"
```
Comment (ravali-rimmalapudi) [inline package.json:83]: Don't we need to install it whenever it is required?

Comment (kridai) [reply package.json:83]: we would dynamically install it  ref:[here](https://github.com/twilio/twilio-cli/blob/main/src/commands/phone-numbers/update.js#L38-L39) if dependency is not present and needed.

Comment (ravali-rimmalapudi) [reply package.json:83]: Great. It make sense now. Thanks :)

**Review by ravali-rimmalapudi — APPROVED** [review]
LGTM

**Pull request merged**